spec/models/member_spec.rb

```diff
@@ -137,4 +137,33 @@
       expect(member.flag_to_organisers?).to be true
     end
   end
+
+  describe '#find_members' do
+    describe 'search by first name' do
+      it "should find the member" do
```

Comment:
The way the other tests (just above this one) express themselves is more confident: `returns the member`. I feel that is clearer, less "perhaps".

```suggestion
      it "returns the member" do
```

Reply:
This rebase was trickier than expected, but I think it's fixed now!